### PR TITLE
add PhysicsSkims to PromptReco

### PR DESCRIPTION
**Pull request by hufnagel**
Deployed in production Tier0 and extensively tested and used there. Only affects PromptReco.

@ticoann , can you wait for the automated test and then just merge if it looks good ?

### CODE REVIEW

**Comment by ticoann** [comment]
retest this please

**Comment by ticoann** [comment]
Hmm test is failing but it seem above test shouldn't be related to this patch.

**Comment by ticoann** [comment]
retest this please

File: src/python/WMCore/WMSpec/StdSpecs/PromptReco.py [hunk 1]
```diff
@@ -19,69 +19,6 @@
...
-    configFile.close()
-
-    scramTempDir = tempfile.mkdtemp()
-    wmcoreBase = getWMBASE()
-    if not envPath:
-        envPath = os.path.normpath(os.path.join(wmcoreBase, "../../../../../../../../apps/wmagent/etc/profile.d/init.sh"))
-    scram = Scram(version = frameworkVersion, architecture = scramArch,
-                  directory = scramTempDir, initialise = initCommand,
-                  envCmd = "source %s" % envPath)
-    scram.project()
```
Comment (ericvaandering) [inline src/python/WMCore/WMSpec/StdSpecs/PromptReco.py]: Actually, re: the e-mail thread, I see at least 4 imports pylint is complaining about are no longer needed when you remove this block of code.

**Comment by ericvaandering** [comment]
FYI if you add a commit to a PR branch you will trigger a retest

Sent from a mobile device. Please excuse my brevity or transcription errors.

On Aug 21, 2015, at 22:48, ticoann <notifications@github.com<mailto:notifications@github.com>> wrote:


retest this please

**Pull request merged**